src/components/nodes/boundaryErrorEvent/boundaryErrorEvent.vue

```diff
@@ -5,9 +5,25 @@
 <script>
 import BoundaryEvent from '@/components/nodes/boundaryEvent/boundaryEvent';
 import errorBoltIcon from '@/assets/boundary-error-event-icon.svg';
+import validBoundaryEventTargets from '@/components/nodes/boundaryEvent/validBoundaryEventTargets';
+import { getAttachedErrorBoundaryEvents } from '@/targetValidationUtils';
 
 export default {
   extends: BoundaryEvent,
+  methods: {
+    doesNotHaveOtherBoundaryEvents(model) {
+      return getAttachedErrorBoundaryEvents(model)
+        .filter(boundaryEvent => {
+          return boundaryEvent !== this.shape;
+        })
```

Comment:
This can be simplified to:
```suggestion
        .filter(boundaryEvent => boundaryEvent !== this.shape)
```